toast-tk-dao/src/main/java/io/toast/tk/dao/domain/impl/report/TestPlanImpl.java

```diff
@@ -6,28 +6,30 @@
 
 import org.bson.types.ObjectId;
 
+import com.github.jmkgreen.morphia.annotations.Embedded;
 import com.github.jmkgreen.morphia.annotations.Entity;
 import com.github.jmkgreen.morphia.annotations.Id;
 import com.github.jmkgreen.morphia.annotations.Index;
 import com.github.jmkgreen.morphia.annotations.Indexes;
 import com.github.jmkgreen.morphia.annotations.Reference;
 
 import io.toast.tk.dao.domain.impl.common.BasicTaggableMongoBean;
+import io.toast.tk.dao.domain.impl.repository.ProjectImpl;
 import io.toast.tk.dao.domain.impl.test.block.ICampaign;
-import io.toast.tk.dao.domain.impl.test.block.IProject;
+import io.toast.tk.dao.domain.impl.test.block.ITestPlan;
 
-@Entity(value = "report.projects")
+@Entity(value = "report.testplans")
 @Indexes({
 		@Index(value = "name"), @Index("version")
 })
-public class Project extends BasicTaggableMongoBean implements IProject {
+public class TestPlanImpl extends BasicTaggableMongoBean implements ITestPlan {
 
 	@Id
 	private ObjectId id;
 
 	private short iteration;
 
-	@Reference
+	@Reference(ignoreMissing=true)
 	private List<ICampaign> campaigns;
 
 	public String version;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make version a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)